.github/workflows/build.yaml

```
@@ -12,29 +12,60 @@ concurrency:
           -Pgpr.user=${{ github.repository_owner }} \
           -Pgpr.token=${{ secrets.GITHUB_TOKEN }}
+
+  dependencies-health:
+    name: Dependencies health
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    steps:
+      - <<: *checkout
```

actionlint (1.7.7)

unexpected key "<<" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)